L1Trigger/L1TMuonBarrel/interface/L1TMuonBarrelParamsAllPublic.h

```diff
@@ -1,6 +1,8 @@
 #ifndef L1TMuonBarrelParamsAllPublic_h
 #define L1TMuonBarrelParamsAllPublic_h
 
+#include <cassert>
```

Comment:
@benkrikler,  @rekovic can you please sign this ( and https://github.com/cms-sw/cmssw/pull/28893/files#diff-d0dc19617f53a94d299fd1f591d10c6fR10 ) for `l1` so that we can proceed with boost 1.72 testing in DEVEL IBs?